WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressFormViewModel.swift

```diff
@@ -134,6 +134,20 @@ final class EditOrderAddressFormViewModel: ObservableObject {
         }
     }
 
+    /// Defines whether "Use as {Shipping/Billing} Address" toggle should be shown.
+    /// Show the toggle, only if the entered address is different from its counterpart.
+    /// i.e. Compare entered address with shipping address while editing billing address, and vice versa.
+    ///
+    var shouldShowUseAddressAsToggle: Bool {
```

Comment:
nit:
```suggestion
    var showUseAddressAsToggle: Bool {
```
just to be more consistent with other similar variables in this class?